.nvmrc

```diff
@@ -0,0 +1 @@
+lts/*
```

Comment:
Why not `lts/carbon`? Then it matches what's specified in `package.json`'s `engines` section.

If we eventually move the minimum required Node version to something newer, we can update both together